src/recommendations/src/recommendations-service/experimentation/experiment_optimizely.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
Please add a license statement at the top of new files. You can copy from one of the other python files.